.github/workflows/CI.yml

```diff
@@ -29,16 +25,21 @@ jobs:
     - name: Set up rustfmt
       run: rustup component add rustfmt
 
+    - name: Create virtualenv
```

Comment:
Why do you create a venv in CI when it's already running on VM ?

Reply:
Because `maturin` requires it, it won't work with system python (seems like a sane choice). Want me to add a comment to explain it ?

Reply:
Oh ok! Didn't know that. Yes please add a comment 🙏